packages/react/src/components/Button/Button.mdx

```diff
@@ -26,39 +32,224 @@ import { Button } from '@digdir/design-system-react';
+
+<br />
+
+### Primary button
+
+Vi bruker denne knappen til den handlingen som er viktigst for brukeren, for eksempel Lagre, Neste eller Send inn. Det skal bare være en primærknapp per side
+
+<Canvas of={ButtonStories.Primary} />
+
+### Secondary button
+
+Vi bruker denne knappen til handlinger som ikke er hovedhandlinger. Den står gjerne sammen med en hovedknapp, så hvis hovedknappen er Lagre, kan sekundærknappen være Avbryt.
+
+<Canvas of={ButtonStories.Secondary} />
+
+### Tertiary button
+
+Denne knappen bruker vi til nedtonede handlinger. Hvis vi vil bruke den alene, må den ha med et ikon, hvis ikke er det vanskelig å se at det er en knapp.
+
+<Canvas of={ButtonStories.Tertiary} />
+
+<br />
+
+## Fargevarianter
+
+### Hovedfarger
+
+Knappene kommer i to hovedfarger `first` og `second`. Hvis du har overstyrt fargepaletten vil fargene `first`og `second` være det du selv har satt.
+
+- `first`-fargen stikker seg mer ut enn `second` og brukes gjerne når handlingen skal ha oppmerksomhet.
+
+<Canvas of={ButtonStories.First} />
+
+- `second`-fargen er mer nøytral og kan brukes når det er flere knapper på en side som ikke skal ta så mye oppmerksomhet.
+
+<Canvas of={ButtonStories.Second} />
+
+### Danger
+
+`Danger`-fargen bruker vi til handlinger som brukeren ikke kan angre, for eksempel Slett.
+
+<Canvas of={ButtonStories.Danger} />
+
+### Success
+
+`Success`-fargen bruker vi bare når brukeren fullfører en prosess.
+
+<Canvas of={ButtonStories.Success} />
+
+### Kombinere ulike fargevarianter
+
+En gruppe med knapper bør bruke samme fargevariant. Unntaket er når vi skal tydeliggjøre en handling brukeren ikke kan angre, for eksempel Forkast. Vi skal ikke kombinere andre fargevarianter.
+
+<Canvas of={ButtonStories.KombinereFarger} />
 
-### Lenke
+<br />
 
-Sett `as="a"` dersom du trenger å bruke knappen som en lenke. Vurder først om det ikke holder å bruke [Link](/docs/felles-link--docs) komponenten.
+## Eksempler
+
+### Knapp som lenke
 
-<Canvas of={ButtonStories.UsedAsLink} />
+Bruk `asChild` dersom du trenger å bruke knappen som en lenke. Vurder først om det ikke holder å bruke [`Link`](/docs/felles-link--docs) komponenten.
+Du kan lese mer om komposisjon [her](https://www.designsystemet.no/grunnleggende/for-utviklere/komposisjon).
 
-### Knapp med tekst
+<Canvas of={ButtonStories.KnappSomLenke} />
 
-<Canvas of={ButtonStories.Variants} />
+### Knapp med kun ikon
 
-### Knapp med ikon
+Knapper med kun ikon bruker vi bare for godt kjente ikoner som Lukk og Slett. Når vi bruker knapper med kun ikon, må vi huske en tilgjengelig tekst som forklarer hva knappen gjør. Dette kan legges i `title`, `aria-label` eller i en `Tooltip` som i eksempelet under.
 
-<Canvas of={ButtonStories.WithIcon} />
+<Canvas of={ButtonStories.BareIkon} />
 
 ### Knapp med tekst og ikon
 
-<Canvas of={ButtonStories.WithIconAndText} />
+Når vi bruker ikon sammen med tekst, skal ikonet ha `aria-hidden` slik at teksten ikke leses opp av skjermleser to ganger.
```

Comment:
Endre til: 
Når vi bruker ikon sammen med tekst, skal ikonet ha `aria-hidden` slik at teksten ikke blir lest opp av skjermlesere to ganger.

Reply:
må vi ha med en tilgjengelig tekst som forklarer hva knappen gjør.